flask_static_digest/__init__.py

```diff
@@ -55,10 +54,19 @@ def _load_manifest(self, endpoint, scaffold):
         except (FileNotFoundError, Exception):
             pass
 
-    def _prepend_host_url(self, host, filename):
-        return urljoin(
-            self.host_url, "/".join([self.static_url_path, filename])
-        )
+    def _custom_url_join(self, url):
+        """
+        This function will join the host URL with another URL.
+        It differs from urljoin in that it will keep the path of the host URL
+        even if url is an absolute path.
+        :param url: the URL returned by url_for
```

Comment:
Can you please change this to: `:param url: The URL returned by url_for`

Reply:
I changed it to url_path as suggested in https://github.com/nickjj/flask-static-digest/pull/38#discussion_r1604143764.